ubiconfig/utils/config_validation.py

```diff
@@ -12,6 +12,6 @@ def validate_config(data, schema=None):
 
     If no schema is provided, :data:`DEFAULT_SCHEMA` is used"""
     if schema is None:
-        with open(DEFAULT_SCHEMA) as f:
-            schema = json.load(f)
+        with open(DEFAULT_SCHEMA) as f:  # pylint: disable=unspecified-encoding
+            schema = json.load(f)  # pylint: disable=unspecified-encoding
```

Comment:
"# pylint: disable=unspecified-encoding" is not required here. The error occurs only for open() function and should be specified only in that line.